File: Front-End/Semana 06/[M1S06] - Ex 7 - Trabalhando com ExportImport/scripts/Pessoa.js
export  class Pessoa{
  get nome() {
    return this._nome;
  }
  set nome(value) {
    this._nome = value;
  }

  get cpf(){
    return this._cpf = cpf;
  }
  set cpf(value){
    this._cpf = value;
  }

  constructor(nome, cpf){
    this._nome = nome;
    this._cpf = cpf;
  };

  imprime(){
    console.log(`${this._nome} - ${this._cpf}`);
  }

};

File: Back-End/projetos de estudo/POO-ASPNET/Entities/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace POO_ASPNET.Entities
{
    public class Pessoa
    {
        public int Id { get; set; }
        public string Nome { get; }

        public virtual void PagarImposto(string tipoImposto)
        {
            Console.WriteLine($"O imposto informado é: {tipoImposto}");
        }
        
        public Pessoa()
        {
            Random random = new Random();
            Id = random.Next(1, 1000);
        }

        public Pessoa(string nome)
        {
            Random random = new Random();
            Id = random.Next(1, 1000);
            Nome = $"{nome}";
        }

    }

}
